src/controllers/handlers/explorer-handler.ts

```diff
@@ -172,13 +175,15 @@ export async function explorerHandler(
     : []
   const trimmedParam = context.url.searchParams.get('trimmed')
   const isTrimmed = trimmedParam === 'true' || trimmedParam === '1'
+  const isSmartWearableParam = context.url.searchParams.get('isSmartWearable')
+  const isSmartWearable = isSmartWearableParam === 'true' || isSmartWearableParam === '1'
```

Comment:
do we need to support the `1` too?

Reply:
I'm supporting 1 too because for the `isTrimmed` we are supporting it as well